src/xERC4626.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.19;
+
+import "solady/src/tokens/ERC4626.sol";
+import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/utils/SafeERC20.sol";
+import "solady/src/utils/SafeCastLib.sol";
+
+abstract contract xERC4626 is ERC4626 {
```

Comment:
Add acknowledgement like `/// @author Modified from Solmate (https://github.com/transmissions11/solmate/blob/main/src/tokens/ERC20.sol)` referencing the tribe repo

Reply:
It wasn't tribe, it was fei. We only need one "modified from" reference to https://github.com/fei-protocol/ERC4626/blob/main/src/xERC4626.sol